client/js/components/statement/assessmentTable/DpSelectedItemsStatements.vue

```diff
@@ -238,7 +237,6 @@ export default {
         }),
         data: payload
       })
-        .then(checkResponse)
         .then(response => {
           const assignee = response.included.find(elem => elem.id === response.data.relationships.assignee.data.id)
           const orgaName = response.included.find(elem => elem.type === 'Claim').attributes.orgaName
```

Comment:
`handleResponseMessages(error.response.data.meta)` in line 258 can be removed (i think)